osf/models/files.py

```diff
@@ -896,7 +896,8 @@ def update_metadata(self, metadata, save=True):
         self.size = self.metadata.get('size', self.size)
         self.content_type = self.metadata.get('contentType', self.content_type)
         if self.metadata.get('modified'):
-            self.date_modified = parse_date(self.metadata['modified'], ignoretz=False)
+            self.metadata['modified'] = str(timezone.now())
```

Comment:
drive by comment-- pretty sure this needs to be ISO 8601 to pass tests and not fail on things.